staging/src/k8s.io/sample-apiserver/README.md

```diff
@@ -24,6 +24,46 @@ HEAD of this repo will match HEAD of k8s.io/apiserver, k8s.io/apimachinery, and
 `sample-apiserver` is synced from https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/sample-apiserver.
 Code changes are made in that location, merged into `k8s.io/kubernetes` and later synced here.
 
+## Normal Build and Deploy
+
+### Changes to the Types
+
+If you change the API object type definitions in any of the
+`pkg/apis/.../types.go` files then you will need to update the files
+generated from the type definitions.  To do this, invoke
+`hack/update-codegen.sh` with `sample-apiserver` as your current
+working directory; the script takes no arguments.
+
+### Build the Binary
+
+With `sample-apiserver` as your current working directory, issue the
+following command.
+
+```
+CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o artifacts/simple-image/kube-sample-apiserver
+```
+
+### Build the Container Image
+
+With `sample-apiserver` as your current working directory, issue the
+following commands with `MYPREFIX` and `MYTAG` replaced by something
+suitable.
+
+```
+docker build -t MYPREFIX/kube-sample-apiserver:MYTAG ./artifacts/simple-image
+docker push MYPREFIX/kube-sample-apiserver:MYTAG
```

Comment:
Can you update the hack/build-image.sh?

Maybe we can add the `go build` command to the script as well.

Reply:
I presume `hack/build-image.sh` is used by some stuff, which would have to be changed if I modified that script to take MYPREFIX and MYTAG as parameters --- but I do not know what that stuff is.  I suggest leaving this to a separate PR.